src/GafferOSL/OSLShader.cpp

```diff
@@ -555,14 +583,18 @@ Plug *loadStructParameter( const OSLQuery &query, const OSLQuery::Parameter *par
 		result = existingPlug;
 	}
 
-	loadShaderParameters( query, result, parameter->name.string() + "." );
+	/// \todo Should we support metadata on the children of a struct parameter?
```

Comment:
Ended up working on other stuff today while waiting for feedback on this.  Do you think I should try to set it up with an underscore?

Note that I can't really think of any scenario where building widgets for a struct parameter is useful.  The main use case for struct parameters would be passing information between nodes, where you would only use the nodule and just hide the widget.